index.html

```diff
@@ -16,11 +17,24 @@
             Aleazar Yilma
         </div>
         <nav class="navigation">
-            <i class="fa-solid fa-bars mobile_element "></i>
+            <div class="mobile_nav show_hide" id="modile_nav">
+                <div class="mobile_nav_close" id="close_mobile_humberger">
+                    <img src="Images/Icon.png" alt=" close menu" class=" mobile_element">
+                </div>
+                <ul class="mobile_menu mobile_element">
+                    <li class="mobile_nav_items"><a href="#">Portoflio</a><img src="Images/icons/rightarrow.png" alt=" Go to Portoflio section"> </li>
+                    <li class="mobile_nav_items" ><a href="#about">About</a><img src="Images/icons/rightarrow.png" alt=" Go to about section"></li>
```

Comment:
- [ ] [optional] Kindly ensure your portfolio and about link , take the user to the correct section of the page, eg about to the about section.